src/constants.tsx

```diff
@@ -64,8 +64,8 @@ export const ScreenSizes = {
     extra_huge_up: '@media only screen and (min-width: 2000px)'
 };
 
-// export const API_URL = 'http://localhost:8080';
-export const API_URL = 'https://pinus-study-backend-deployment-production.up.railway.app';
+export const API_URL = 'http://localhost:8080';
+//export const API_URL = 'https://pinus-study-backend-deployment-production.up.railway.app';
```

Comment:
Isn't it better to leave it on railway?
```suggestion
// export const API_URL = 'http://localhost:8080';
export const API_URL = 'https://pinus-study-backend-deployment-production.up.railway.app';
```